pages/team.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: page
+title: MDAnalysis team
+---
+
+MDAnalysis is a community-driven project that is made possible through the efforts of many members who contribute in numerous and diverse ways, ranging from direct package development, maintenance, documentation, communication, and managerial responsibilities. On this page we list identified project roles and those historically responsible for the tasks. In the future, we plan to also list the contributors currently responsible for overseeing and managing associated tasks. We note that the listed roles on this page can differ significantly in scope and required effort.
+
+## Roles
+
+In this section we list contributors who have led or contributed substantial effort to the roles listed below.
+At present, we have only listed leading figures from the past 1-2 years, i.e. from 2021 onwards.
+However, we hope to eventually acknowledge everyone who has been responsible for a particular role throughout
+the lifetime of the project.
+
+If you have participated to MDAnalysis in the past and feel your previous/historical role could be better characterised,
+please contact us at mdanalysis@numfocus.org to let us know!
+
+| Role                         | Sub-role                       | Historical                                                              |
```

Comment:
One of the main questions I have about this table is "how will this look tomorrow" - i.e. how will this table change to include "current" folks in the next update?

Mostly thinking about how the information flow will make sense when that happens.

Reply:
My thought was to just add a "current people" column, and highlight that as separate from the historic people, but that may not be the most ideal?

Reply:
Yeah I'm sorta thinking "what if historic is current" - which I guess is fair, it just makes history complicated.

Reply:
The other question I had was about leads & such but I think that can wait.

Reply:
My eventual vision was as Fiona mentioned, a table with the headers | Role | Subrole | Lead | Subgroup | Historical | (historical at the end). 

I agree it could be messy to have repeated names and I was thinking of removing names from "historical" if they were also current. We could also leave them in and just keep the messiness. Unless we start defining things even further by specific periods of time, I'm not sure how to avoid either scenario.

Reply:
It doesn't have to be dealt with immediately, it was mostly a curiosity about how the table will eventually look. At the end of the day something is better than nothing.

Reply:
Another option is to have "historical" as a totally separate section, but that could be repetitive.